checkpoint_quantum_firewall/README.md

```diff
@@ -0,0 +1,132 @@
+## Overview
+
+Check Point Next Generation Firewall is a security gateway that includes application control and IPS protection, with integrated management of security events. Additional features include Identity Awareness, URL Filtering, Anti-Bot, Anti-Virus, and Anti-Spam.
+
+## Setup
+
+### Installation
+
+To install the Checkpoint Quantum Firewall integration, follow the steps below:
+
+1. [Install][5] the 1.0 release (`checkpoint_quantum_firewall==1.0.0`).
+
+### Configuration
+
+#### Log collection
+
+**Checkpoint Quantum Firewall:**
+
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in the `datadog.yaml` file:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Add this configuration block to your `checkpoint_quantum_firewall.d/conf.yaml` file to start collecting your Checkpoint Quantum Firewall logs.
```

Comment:
Same comment as in https://github.com/DataDog/integrations-core/pull/16499/files#r1449968809

Reply:
Done